### PR TITLE
Backend - Dev Tools - Migration Generation

### PR DESCRIPTION
**Pull request by Mekhi-M**
This PR was made to add a command that will make our lives easier.

I added the `"migrate:make"` command into our package.json so that we can run something like `npm run migrate:make user_posts_table` from the root directory. 
> Without this command we would have to cd into our server first before we run `npx knex migrate:make file_name` Of course, you can still do that if you want... but I just made it easy!!!

With this we can easily create new migration files instead of using the `migrate:rollback` command. Since we were advised against using the `migrate:rollback` command to create/edit our tables lets just follow the suggestion and use this command instead

TL;DR
DO NOT `npx knex migrate:rollback`
DO `npm run migrate:make file_name`

### CODE REVIEW

**Comment by PatrickDacius** [comment]
That sounds good, seems like this would save us from some unnecessary problems in the near future
